src/seq/bioseq.jl

```diff
@@ -237,6 +237,16 @@ alphabet{A}(::Type{BioSequence{A}}) = alphabet(A)
 Base.length(seq::BioSequence) = length(seq.part)
 Base.eltype{A}(::Type{BioSequence{A}}) = eltype(A)
 
+function Base.count(f::Function, seq::BioSequence)
+    n = 0
+    for x in seq
+        if f(x)
+            n += 1
+        end
```

Comment:
It may be. But I think the Julia compiler is not so smart to parallelize the loop because it contains a function call, which will not be inlined as far as I know.